#!/bin/sh /usr/share/dpatch/dpatch-run
## 02_libbrlapi_fix.dpatch by Luke Yelavich <themuso@ubuntu.com>
##
## DP: Change the brltty library file that gets opened in brlmodule.c

@DPATCH@
diff -Nurp orca-0.2.8/src/brl/brlmodule.c orca-0.2.8.new/src/brl/brlmodule.c
--- orca-0.2.8/src/brl/brlmodule.c	2006-06-04 05:16:00.000000000 +1000
+++ orca-0.2.8.new/src/brl/brlmodule.c	2006-08-17 15:59:58.000000000 +1000
@@ -115,11 +115,11 @@ static PyObject *brl_module_init(PyObjec
 
         /* Open the brlapi library
          */
-        brlapi_library = dlopen("libbrlapi.so", RTLD_LAZY);
+        brlapi_library = dlopen("libbrlapi.so.0.4", RTLD_LAZY);
         if (!brlapi_library) {
                 PyErr_SetString(
                         PyExc_StandardError,
-                        "Failed to load libbrlapi.so");
+                        "Failed to load libbrlapi.so.0.4");
                 return NULL;
         }
